sr_tactile_sensors/launch/sr_mst_hand_plotjuggler_visualiser.launch

```diff
@@ -0,0 +1,20 @@
+<!--
+ Copyright 2024 Shadow Robot Company Ltd.
+ This program is free software: you can redistribute it and/or modify it
+ under the terms of the GNU General Public License as published by the Free
+ Software Foundation version 2 of the License.
+ This program is distributed in the hope that it will be useful, but WITHOUT
+ ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ more details.
+ You should have received a copy of the GNU General Public License along
+ with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<launch>
+  <!-- Hand and sensor configurations-->
+  <arg name="hand_id" default="rh"/>
+
+  <node name="$(arg hand_id)_MST_plotjuggler_visualiser" pkg="plotjuggler" type="plotjuggler" args="--layout $(find sr_tactile_sensors)/config/hand_plotjuggler_config_$(arg hand_id).xml"/>
```

Comment:
We probably should add `plotjuggler` as a `<run_depend>` to the package.xml

Reply:
good point. Shouldn't CI have caught that?